ls1intum/Artemis
# Assessment Queue for Automatic Text Assessment

### Checklist

### Motivation and Context
It decides which submission should be assessed by the TA/Instructor next, to make the Automatic Assessment better

### Description
For non automatic text  assessments, the next submission is chosen randomly. For automatic assessments it calculates the information gain, which describes the value a submission has for the automatic assessment. A submission is better when it's textblocks have many similar textblocks and are assigned to large clusters

### Steps for Testing

1. Have a Course for which automatic assessment is enabled:
1.1 Create a new Course and check the automatic assessment checkbox and set the due date in 10 minutes
1.2 Submit larger chunks of real text with 5-10 Accounts, with similar content
1.3 Wait until the due date to get the assessments clustered
2. go to the tutor dashboard
3. check if a submission is proposed to you
